test/integration/lib/config/foxbox.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+
+	nupnp_server : {
```

Comment:
It doesn't appear in the last commits' changes. Maybe you need to make a modification within the file, so the IDE changes all the rest?